Resources/doc/security/index.md

```diff
@@ -6,5 +6,6 @@ Security
 * [Fields public control](fields-public-control.md)
 * [Limiting query depth](limiting-query-depth.md)
 * [Query complexity analysis](query-complexity-analysis.md)
+* [Disable introspection](disable_introspection.md)
```

Comment:
please also add this entry to main readme file.

Reply:
will do